kitsune/settings.py

```diff
@@ -602,12 +604,17 @@ def _username_algo(email):
         FXA_USE_NONCE = config("FXA_USE_NONCE", False)
         FXA_LOGOUT_REDIRECT_URL = config("FXA_LOGOUT_REDIRECT_URL", "/")
         FXA_USERNAME_ALGO = config("FXA_USERNAME_ALGO", default=_username_algo)
-        FXA_STORE_ACCESS_TOKEN = config("FXA_STORE_ACCESS_TOKEN", default=False, cast=bool)
+        FXA_STORE_ACCESS_TOKEN = config("FXA_STORE_ACCESS_TOKEN", default=True, cast=bool)
         FXA_STORE_ID_TOKEN = config("FXA_STORE_ID_TOKEN", default=False, cast=bool)
         FXA_SUBSCRIPTIONS = config(
             "FXA_SUBSCRIPTIONS", default="https://accounts.firefox.com/subscriptions"
         )
         FXA_SET_ISSUER = config("FXA_SET_ISSUER", default="https://accounts.firefox.com")
+        FXA_VERIFY_URL = config("FXA_VERIFY_URL", default="")
```

Comment:
Should this have a default value?

Reply:
I will add prod to match the rest of the vars.